### PR TITLE
feat(umaverse): add first page for v2

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**

This PR adds the first page for Umaverse v2!

**Details**

All data for now is mocked, along and search and navigation are not expected to work. 

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,24 +1,141 @@
...
+                  </>
+                )}
+              />
+            </CardContent>
+          </Card>
+          <Card>
+            <InfoIcon />
+            <CardContent>
+              <CardHeading>
+                TVM <span>(Total Value Minted)</span>
```
Comment (Tommy1231232) [inline pages/index.tsx:44]: For these headers, can you confirm they are the same size and font in the Figma? Looks slightly different

Comment (Tommy1231232) [reply pages/index.tsx:44]: I don't' think what's in parenthesis should be bolded

Comment (Tommy1231232) [reply pages/index.tsx:44]: I think for these headers it's better to have Total Value Locked (TVL) versus TVL (Total Value Locked)

File: components/SearchBar.tsx [hunk 2]
```diff
@@ -1,35 +1,44 @@
...
-  const [query, setQuery] = useState("");
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setQuery(event.target.value);
-  };
   return (
     <Wrapper>
-      <Label>Search for synths</Label>
-      <Input onChange={handleChange} placeholder="uStonks" value={query} />
       <Icon />
+      <Input placeholder="Search for projects..." />
```
Comment (Tommy1231232) [inline components/SearchBar.tsx:9]: Align to figma (same color gray and size with "Search project"

File: components/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styled from "@emotion/styled";
```
Comment (Tommy1231232) [inline components/Button.tsx:1]: Ignore the location of this comment, but when a user clicks a category the rank should start from 1. Right now if you click a category it pulls in a synth that has #3 rank in the first spot. Lets make this be #1, #2, etc,

Comment (Tommy1231232) [reply components/Button.tsx:1]: -Can't comment exactly where this is in the codebase but the design for the rank should match to figma. There is a gray circle in the design. 

-Under each synth, unbold and grey the text under the synth name (similar to figma)

-Unbold category names next to each synth and "only show live projects)

Do a general check against figma for things in bold

Comment (Tommy1231232) [reply components/Button.tsx:1]: There should be more space between the search bar and the top of the black banner. As well as between "Explore the UMAverse"

https://www.figma.com/file/JicBZVZCFFRbOESAC3EEVc/UMA---UMAverse-v2?node-id=0%3A1

Comment (Tommy1231232) [reply components/Button.tsx:1]: Unbold the words right below "Explore the UMAverse"

Comment (Tommy1231232) [reply components/Button.tsx:1]: The TVL, TVM and % change boxes look a little too wide compared to the designs

File: pages/index.tsx (hunk 1 above)
Comment (Tommy1231232) [reply pages/index.tsx:44]: Clicking the UMA logo in the top left corner should take the user back to the main page of UMAverse instead of umaproject.org

**Review by daywiss — APPROVED** [review]
looks great to me, responsive and cool animations on table

**Pull request merged**